source/adios2/toolkit/format/bp5/BP5Serializer.cpp

```diff
@@ -600,14 +599,14 @@ static void GetMinMax(const void *Data, size_t ElemCount, const DataType Type,
     if (Type == DataType::Struct)
     {
     }
-#ifdef ADIOS2_HAVE_CUDA
+#ifdef ADIOS2_HAVE_GPU_SUPPORT
 #define pertype(T, N)                                                          \
-    else if (MemSpace == MemorySpace::CUDA &&                                  \
+    else if (MemSpace != MemorySpace::Host &&                                  \
```

Comment:
```suggestion
    else if (MemSpace == MemorySpace::GPU &&                                  \
```

Reply:
The initial code was using multiple backend so `!=` made sense. For the current code version it's the same thing. If you have a preference for equal, I can change it